task_interperator_ipf_client/__manifest__.py

```diff
@@ -32,15 +32,15 @@
     'author': "N-development",
     'license': 'AGPL-3',
     'website': 'https://www.n-development.com',
-    "depends": [
-        'mail',
+    "depends": ['res_interpreter_language',
+                'mail',
     ],
     'data': [
         "security/ir.model.access.csv",
         'views/client_config_views.xml',
         'views/res_config_settings_views.xml',
     ],
-    'installable': True,
+    'installable': False,
```

Comment:
If this module is something depricated, state it in the name field.
Like `'name': 'DEPRICATED - Outplacement Intepretor IPF Client'`